rest/server_context.go

```diff
@@ -1010,6 +1013,7 @@ func (sc *ServerContext) asyncDatabaseOnline(nonCancelCtx base.NonCancellableCon
 	// Before bringing the database online, ensure that the database hasn't been modified while we waited for initialization to complete
 	currentDbVersion := sc.GetDbVersion(dbc.Name)
 	if currentDbVersion != version {
+		dbc.DbStats.DatabaseStats.TotalOnlineFatalErrors.Add(1)
```

Comment:
I don't think this is a fatal error, the newer version is going to get picked up?